### ISSUE TITLE
Is "user responsible for paying" always going to be a new party added to the case?

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-EFSPIntegration#69]
Currently the only options are in the `users` list and we set the value to a string.  Just wanted to make sure that is correct.

**Comment by BryceStevenWilley** [comment]
No, we should have the ability to add others!

For my own reference, if the party is in the current filing the XML needs to look like this:
```
<ecf:FilingPartyID>
<nc:IdentificationID>Party1</nc:IdentificationID>
<nc:IdentificationCategoryText>REFERENCE</nc:IdentificationCategoryText>
</ecf:FilingPartyID>
```
Where "Party1" is the XML id on that CaseParticipant.

And if they are already a part of the case, it needs to look like this:
```
<ecf:FilingPartyID>
<nc:IdentificationID>037BAEC1-F6E8-43ED-A102-B8C5E76748EC</nc:IdentificationID>
<nc:IdentificationCategoryText>IDENTIFICATION</nc:IdentificationCategoryText>
</ecf:FilingPartyID>
```
Where the ID is the EFM identifier of the case participant, which we can get from the GetCase call or a GetFiling call for the previous filing.

**Comment by nonprofittechy** [comment]
See #125

**Comment by nonprofittechy** [comment]
This may be mostly resolved by the work in #134, @BryceStevenWilley. I'll leave to you to decide when it's ready to close though
